src/Time/TimeSteppers/TimeStepper.hpp

```diff
@@ -65,6 +65,23 @@ class TimeStepper : public PUP::able {
         this, u, history, time_step);
   }
 
+  /// Add the change for the current substep to u; report the error measure when
+  /// available.
+  ///
+  /// For a substep method, the error measure will only be available on full
+  /// steps. For a multistep method, the error measure will only be available
+  /// when a sufficient number of steps are available in the `history` to
+  /// compare two orders of step. Whenever the error measure is unavailable,
+  /// `u_error` is unchanged and the function return is `false`.
+  template <typename Vars, typename DerivVars>
+  bool update_u(
```

Comment:
[picking line at random]: should the steppers report the order of the method used for the error estimate or are we requiring it always be one order lower? I ask because if one wants to decide on the optimal step size that might would differ depending on how the error is computed

Reply:
That's a fair point -- it's probably good to report the order of the error computation. Currently all of the steppers report errors based on estimates of exactly one lower order, but that's not strictly required, I think.

Reply:
Actually, sorry, I'll address this in a future PR -- for the step chooser changes I need to add order reporting anyway (steppers currently don't report their stepper order either), so the error order report is more natural in that set of changes, I think.

Reply:
Okay, sounds good to me!